src/record/delete/usecases/delete.ts

```diff
@@ -1,49 +1,92 @@
 import type { KintoneRestAPIClient } from "@kintone/rest-api-client";
```

Comment:
@mshrtsr 

I agree with you, the name of this file should contain `ByRecordNumber` or `ByFile`.

But, I prefer to use `ByFile`. Because, if we use `ByRecordNumber`, It maybe make confusion when the option is `--file-path`

What do you think?

NOTE: the full file name is `deleteByFile.ts` or `deleteByRecordNumber.ts`

Reply:
Yes, I think `deleteByRecordNumber` is better for the same reason as the above comment! :)